config/Dockerfiles/tests.d/beaker/02_beaker_distro

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -xe
+
+# Verify distro selection in beaker
+# distros.exclude: none
+# providers.include: beaker
+# providers.exclude: none
+
+DISTRO=${1}
+PROVIDER=${2}
+
+PINFILE="PinFile.${PROVIDER}.yml"
+DTARGET="beaker-distro"
+FTARGET="beaker-family"
+TMP_FILE=$(mktemp)
+
+function clean_up {
+    set +e
+    linchpin -w . -p "${PINFILE}" -v destroy "${DTARGET}" "${FTARGET}"
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+pushd docs/source/examples/workspace
+linchpin -w . -p "${PINFILE}" -v up "${DTARGET}" "${FTARGET}"
```

Comment:
How is this test different from https://github.com/p3ck/linchpin/blob/23729fbe23f430fb0e9f4c4e28a5b1bc59009a77/config/Dockerfiles/tests.d/general/01_general ?

Reply:
@p3ck nvm, I see the target data.